setup.py

```diff
@@ -14,6 +14,7 @@
 
 install_requires = [
     'draftjs_exporter==0.7.0',
+    'jinja2>=2.9.5,<3'  # TODO: remove when it's not necessary (https://github.com/springload/wagtaildraftail/issues/13)
```

Comment:
Wagtail's `rich_text` filter [does use mark_safe](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailcore/templatetags/wagtailcore_tags.py#L53-L62) so the `draft_text` should do the same.